src/customConfigTests/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1 @@
+org.slf4j.simpleLogger.defaultLogLevel=off
```

Comment:
making the output logs much quieter so that we don't get e.g. massive messages from hoverfly about mismatched calls